profiles/scalingo.json

```diff
@@ -1,10 +1,10 @@
 {
   "name": "Scalingo",
-  "revision": "2015-07-22",
-  "vendor_verified": "2015-04-17",
+  "revision": "2016-08-04",
+  "vendor_verified": "2015-02-23",
   "url": "https://scalingo.com",
   "status": "production",
-  "status_since": "2015-04-17",
+  "status_since": "2015-02-03",
```

Comment:
`status_since` means the date since Scalingo is in `production` Why did you change this?